heltour/tournament/views.py

```diff
@@ -1801,6 +1801,54 @@ def view(self, post=False):
     def view_post(self):
         return self.view(post=True)
 
+class ConfirmScheduledTimeView(LeagueView, LoginRequiredMixin):
```

Comment:
as a general rule of thumb with multiple inheritance, you want to go mixins first, then base classes. there are few edge cases where this matters, not sure if this is one of them

```suggestion
class ConfirmScheduledTimeView(LoginRequiredMixin, LeagueView):
```